.github/workflows/ci.yml

```diff
@@ -120,14 +120,13 @@ jobs:
     needs: build
     runs-on: ubuntu-latest
     if: "!github.event.pull_request.head.repo.fork"
-    timeout-minutes: 15
     strategy:
-      max-parallel: 2
+      max-parallel: 3
       fail-fast: false
       matrix:
         terraform:
           - '1.8.*'
-          - '1.9.*'
+#          - '1.9.*'
```

Comment:
Will we add this back when Terraform 1.10 comes out?

Reply:
yes